advanced/databases/mysql-is/requirements.yaml

```diff
@@ -13,5 +13,5 @@
 # limitations under the License.
```

Comment:
@vordimous please change the target branch to `5.11.x` as it is the development branch.